src/utils/customDocumentLoader.ts

```
@@ -0,0 +1,49 @@
+import type { AgentContext, DocumentLoader } from '@credo-ts/core'
+
+import { CredoError } from '@credo-ts/core'
+import { DocumentLoaderResult } from '@credo-ts/core/build/modules/vc/data-integrity/jsonldUtil'
```

ESLint

[error] All imports in the declaration are only used as types. Use `import type`.

(@typescript-eslint/consistent-type-imports)